books/free-programming-books-ar.md

```diff
@@ -8,6 +8,7 @@
 * [HTML/CSS](#htmlcss)
 * [Introduction to Programming in Arabic](#introduction-to-programming-in-arabic)
 * [JavaScript](#javascript)
+  * [Next.js](#next.js)
```

Comment:
```suggestion
* [JavaScript](#javascript)
```

Reply:
I can't commit this suggestion, github disable the buttons